index.html

```diff
@@ -1311,19 +1292,6 @@
   choices.</span>
 </div>
 
-<aside class="example">
-  Sites and browsers wish to collect telemetry data to determine how frequently features are used or
-  to debug breakages, but the user agent does not want to burden the user with frequent consent
-  requests. A browser could use a first-run dialog to ask the user whether they generally support
-  sharing data to find bugs and improve the Web software they use, and then enable or disable
-  telemetry and reporting APIs based on the user's choice.
-</aside>
```

Comment:
When the group was discussing this topic, this example had what I believe was strongest consensus for it, in documenting how we wanted to see general preferences solicited in a way that would avoid unnecessary prompts and labor.